### ISSUE TITLE
return from nested ifs woes in func

### ISSUE THREAD

**Issue by kelvich** [ton-blockchain/ton#85]
Running following code with id = 30 returns 60 90 5 6 instead of 60 90.

```
(int, int) nested_if(int id) method_id {
  dump_stack();
  if (id > 0) {
    if (id > 10) {
      return (2 * id, 3 * id);
    }
  }
  return (5, 6);
}
```
run trace:

```
execute SETCP 0
execute DICTPUSHCONST 19 (xC_,1)
execute DICTIGETJMP
execute DUMPSTK
#DEBUG#: stack(1 values) : 30 
execute DUP
execute GTINT 0
execute PUSHCONT x20C20A9620AA0001A703E030
execute PUSHCONT x30
execute IFELSE
execute DUP
execute GTINT 10
execute PUSHCONT x20AA0001A703
execute IFJMP
execute DUP
execute LSHIFT 1
execute SWAP
execute MULINT 3
execute implicit RET
execute PUSHINT 5
execute PUSHINT 6
execute implicit RET
[ 3][t 0][1570173908.227529049][continuation.cpp:772]	steps: 21 gas: used=714, max=9223372036854775807, limit=9223372036854775807, credit=0
60 90 5 6 0 C{3FE93897158698E4D473B74414D7493716B0FC3A70310934873F0019DAACCAB4} 
```

**Comment by ton-blockchain** [comment]
Yes, if one if-else branch in funC code returns, the other should return, too. Otherwise funC compiles something different from what you'd expect. In your example this condition is violated for the inner if.

Maybe we'll change the funC compiler to output an error instead.

**Comment by ton-blockchain** [comment]
Partially "fixed" in commit f67f5d879ba0a395367a703dcd75f3b87c6ad535, in that funC compiler will generate an error instead of compiling incorrect code. However, `return`s inside loops would still lead to incorrect code (to be fixed in a separate future commit).
